### PR TITLE
Star service

### PR DESCRIPTION
**Pull request by MaxNoe**
To get rid of hardcoded zeta tauri, I use the yale bright star catalogue to look for bright stars in the FOV.

Tests show that Zeta Tauri is found for crab pointing positions.
I set the limit to a magnitude of 4.0 for the analysis, so for crab the behaviour is not changed.

This also contains two fixes for the HDUReader and a feature addition:
* Addressing HDUs by index is now possible
* HeaderLines are now correctly parsed if multiple slashes are present
* Lines without key/value pair are ignored
* Duplicate keys are ignored.

### CODE REVIEW

File: src/main/java/fact/cleaning/TwoLevelTimeMedian.java [hunk 1]
```diff
@@ -55,10 +53,10 @@
     public int minNumberOfPixel;
 
     @Parameter(required = false)
-    public String[] starPositionKeys = null;
+    public String starPositionsKey = null;
 
     @Parameter(required = false, defaultValue = "Constants.PIXEL_SIZE_MM")
-    public double starRadiusInCamera = Constants.PIXEL_SIZE_MM;
+    public double starRadiusInCamera = 11.0;
```
Comment (jebuss) [inline src/main/java/fact/cleaning/TwoLevelTimeMedian.java]: why is this number changed?

File: src/main/java/fact/cleaning/TwoLevelTimeMedian.java [hunk 2]
```diff
@@ -55,10 +53,10 @@
     public int minNumberOfPixel;
 
     @Parameter(required = false)
-    public String[] starPositionKeys = null;
+    public String starPositionsKey = null;
```
Comment (jebuss) [inline src/main/java/fact/cleaning/TwoLevelTimeMedian.java]: does this change the meaning of this key? Could you add a doc string?

File: src/main/java/fact/cleaning/TwoLevelTimeNeighbor.java [hunk 3]
```diff
@@ -59,12 +57,11 @@
             " If Size is smaller than minSize the Pixels will be discarded.")
     public int minNumberOfPixel;
 
-
     @Parameter(required = false)
-    public String[] starPositionKeys = null;
+    public String starPositionsKey = null;
```
Comment (jebuss) [inline src/main/java/fact/cleaning/TwoLevelTimeNeighbor.java]: doc string?

File: src/main/java/fact/cleaning/TwoLevelTimeNeighbor.java [hunk 4]
```diff
@@ -59,12 +57,11 @@
...
 
     @Parameter(required = false, defaultValue = "Constants.PIXEL_SIZE")
-    public double starRadiusInCamera = Constants.PIXEL_SIZE_MM;
+    public double starRadiusInCamera = 11.0;
```
Comment (jebuss) [inline src/main/java/fact/cleaning/TwoLevelTimeNeighbor.java]: s.o.

File: src/main/java/fact/starservice/Star.java [hunk 5]
```diff
@@ -0,0 +1,22 @@
+package fact.starservice;
+
+import fact.coordinates.EquatorialCoordinate;
+
+public class Star implements  Comparable<Star> {
```
Comment (jebuss) [inline src/main/java/fact/starservice/Star.java]: Could you add a docstring about the class?

File: src/main/java/fact/starservice/StarService.java [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+import fact.io.hdureader.FITS;
+import fact.io.hdureader.OptionalTypesMap;
+import stream.service.Service;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class StarService implements Service {
```
Comment (jebuss) [inline src/main/java/fact/starservice/StarService.java]: Docstring

File: src/main/java/fact/starservice/StarsInFOV.java [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+/**
+ * This calculates the position of the source in the camera.
+ *
+ * @author Maximilian Noethe &lt;maximilian.noethe@tu-dortmund.de&gt;
+ */
+
+public class StarsInFOV implements Processor {
+
+
+    private static final Logger log = LoggerFactory.getLogger(fact.features.source.SourcePosition.class);
```
Comment (jebuss) [inline src/main/java/fact/starservice/StarsInFOV.java]: copy paste artifact? `getLogger(fact.features.source.SourcePosition.class);`

File: src/main/java/fact/cleaning/TwoLevelTimeMedian.java (hunk 2 above)
Comment (MaxNoe) [reply src/main/java/fact/cleaning/TwoLevelTimeMedian.java]: done

File: src/main/java/fact/cleaning/TwoLevelTimeNeighbor.java (hunk 3 above)
Comment (MaxNoe) [reply src/main/java/fact/cleaning/TwoLevelTimeNeighbor.java]: done

File: src/main/java/fact/starservice/Star.java (hunk 5 above)
Comment (MaxNoe) [reply src/main/java/fact/starservice/Star.java]: done

File: src/main/java/fact/cleaning/TwoLevelTimeMedian.java (hunk 1 above)
Comment (jebuss) [reply src/main/java/fact/cleaning/TwoLevelTimeMedian.java]: placet

**Pull request merged**